server/app/models/ProgramTab.java

```diff
@@ -0,0 +1,26 @@
+package models;
+
+/**
+ * Represents the tabs available to Civiform admins in the program list.
+ *
+ * <p>Tabs:
+ *
+ * <ul>
+ *   <li>{@link #IN_USE}: Displays a list of programs currently in use.
+ *   <li>{@link #DISABLED}: Displays a list of disabled programs.
+ * </ul>
+ */
+public enum ProgramTab {
+  IN_USE("In use"),
+  DISABLED("Disabled");
+
+  private final String tab;
+
+  ProgramTab(String tab) {
+    this.tab = tab;
+  }
+
+  public String getProgramTab() {
```

Comment:
suggestion (non-blocking): I see some of the other enum classes calling this `getValue` so you might want to follow that convention, but I don't feel strongly.

Reply:
Could also call it what it is, which is `getTabName()` or `getUILabel()`.

In this case I'd also recommend renaming the `tab` property to be `tabName` or `uiLabel` for consistency.